connections/management/commands/send_oikotie_xml_file.py

```diff
@@ -32,16 +32,25 @@ def handle(self, *args, **options):
         if not options["only_create_files"] and apartment_file and housing_file:
             try:
                 send_items(apartment_file)
+                _logger.info(
+                    f"Succefully sent XML file {apartment_file} to Oikotie FTP server"
+                )
+            except Exception as e:
+                _logger.error(
+                    f"File {apartment_file} sending via FTP to Oikotie failed:",
+                    str(e),
+                )
+                raise e
+            try:
                 send_items(housing_file)
                 _logger.info(
-                    f"Succefully sent XML file {apartment_file} and \
-{housing_file} to Oikotie FTP server"
+                    f"Succefully sent XML file {housing_file} to Oikotie FTP server"
                 )
             except Exception as e:
                 _logger.error(
-                    f"Oikotie XML files {apartment_file} and {housing_file} not sent:",
+                    f"File {housing_file} sending via FTP to Oikotie failed:",
                     str(e),
                 )
                 raise e
         else:
-            _logger.info("Oikotie XML files not sent")
+            _logger.info("Not sending XML files to Oikotie")
```

Comment:
Felt like should leave log here as if there are empty files, there is not need to send them but maybe still good to indicate that in log